### ISSUE TITLE
The robot will fly away in gazebo in bring_up demo

### ISSUE THREAD

**Issue by Tianhuanyu** [lbr-stack/lbr_fri_ros2_stack#129]
When I run this demo:
`
ros2 launch lbr_bringup bringup.launch.py model:=iiwa7 sim:=true rviz:=true moveit:=true

`
I meet this issue:

https://youtu.be/Muy3wggZf3s

The planning execution of robot should have no problem, but the basement link may be not fixed. So the robot 'fly away' in gazebo

**Comment by mhubii** [comment]
thanks for sharing this issue @Tianhuanyu! This was introduced with #121. We are investigating a fix

**Comment by mhubii** [comment]
when launching the system with `base_frame:=world`, everything works as intended. E.g.

```shell
ros2 launch lbr_bringup bringup.launch.py model:=iiwa7 moveit:=true sim:=true base_frame:=world
```

It appears that Gazebo requires a specific name for the base frame.

**Comment by mhubii** [comment]
fixed in #144
